### PR TITLE
(SIMP-3150) Standardize cron data types

### PR DESCRIPTION
**Pull request by sharkbruhaha**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,6 @@
+* Mon Mar 25 2019 Joseph Sharkey <shark.bruhaha@gmail.com> - 6.2.0-0
+- Standardized cron datatypes to use the Simplib::Cron::### types
```
Comment (lnemsick-simp) [inline CHANGELOG]: Why is this a minor version bump? I would think this is a patch bump. It is completely backward compatible and simply allows more flexibility in values allowed for cron-related parameters.

Comment (sharkbruhaha) [reply CHANGELOG]: It was new functionality for those fields so I wasn't sure what the cutoff for a patch bump was.

Comment (sharkbruhaha) [reply CHANGELOG]: It is not backwards compatible because the new datatype allows for Integers, not just Strings.

**Comment by lnemsick-simp** [comment]
https://gitlab.com/simp/pupmod-simp-clamav/pipelines/53903867

**Comment by lnemsick-simp** [comment]
https://gitlab.com/simp/pupmod-simp-clamav/pipelines/53905483

**Pull request merged**